lv_conf_template.h

```diff
@@ -1076,11 +1076,14 @@
 #define LV_USE_FONT_MANAGER                     0
 #if LV_USE_FONT_MANAGER
 
-/*Font manager name max length*/
+/**Font manager name max length*/
 #define LV_FONT_MANAGER_NAME_MAX_LEN            32
```

Comment:
It is okay to make this symbol hard limited instead to expose it into lv_conf and if not defined there then default to the value above?

Reply:
For the sake of simplicity [lv_conf_internal.h](https://github.com/lvgl/lvgl/blob/master/src/lv_conf_internal.h) defines everything to a default value. So in the end there is no such a thing as a on-defined config